### PR TITLE
Update rubygem-jquery-ui-rails to 6.0.1

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Left the dependencies in foreman untouched. Is that correct? I'm still unsure on guidance for that.

### CODE REVIEW

**Comment by mmoll** [comment]
The railties version changed (3.1.0 -> 3.2.16), so this needs to get reflected in the spec, too.

**Comment by ekohl** [comment]
I would recommend regenerating the spec using `./add_gem_package.sh`. That includes the right content so `./bump_rpm.sh` can pick up the changes in dependencies.

File: packages/foreman/rubygem-jquery-ui-rails/rubygem-jquery-ui-rails.spec [hunk 1]
```diff
@@ -1,75 +1,95 @@
...
-Requires: %{?scl_prefix}rubygem(railties) >= 3.1.0
+Requires: %{?scl_prefix_ruby}ruby(rubygems) >= 1.3.6
+Requires: %{?scl_prefix}rubygem(railties) >= 3.2.16
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel >= 1.3.6
 BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel >= 1.3.6
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
```
Comment (ezr-ondrej) [inline packages/foreman/rubygem-jquery-ui-rails/rubygem-jquery-ui-rails.spec:24]: Is the `Obsolete` still needed?

**Comment by ezr-ondrej** [comment]
@ekohl I did `./add_gem_package` and the changes seems ok I'd guess, just the compilation is unecessary should I remove that? Does it look ok to you?

File: packages/foreman/rubygem-jquery-ui-rails/rubygem-jquery-ui-rails.spec [hunk 2]
```diff
@@ -1,75 +1,95 @@
...
 
-rm -rf %{buildroot}%{gem_instdir}/{.gitignore,.gitmodules,.travis.yml,Gemfile,Rakefile}
-
-
-
 %files
 %dir %{gem_instdir}
-%{gem_libdir}
+%exclude %{gem_instdir}/.gitignore
+%{gem_instdir}/.gitmodules
```
Comment (ekohl) [inline packages/foreman/rubygem-jquery-ui-rails/rubygem-jquery-ui-rails.spec:73]:
```suggestion
%exclude %{gem_instdir}/.gitmodules
```

File: packages/foreman/rubygem-jquery-ui-rails/rubygem-jquery-ui-rails.spec (hunk 1 above)
Comment (ekohl) [reply packages/foreman/rubygem-jquery-ui-rails/rubygem-jquery-ui-rails.spec:24]: Not really, we've started to drop them in other packages as well.

**Comment by ekohl** [comment]
@zjhuntin @pcreech the JSON repoclosure issue looks unrelated. Known issue?

**Comment by ezr-ondrej** [comment]
> @zjhuntin @pcreech the JSON repoclosure issue looks unrelated. Known issue?

It seems to be issue introduced by #5021

**Comment by pcreech** [comment]
[test rpm]

**Comment by pcreech** [comment]
@ekohl I'm not seeing that json repoclosure among the repoclosure failures (i reran it due to the job being garbage collected).  Judging by the state of the current repoclosure failures, looks like we are currently in an inconsistent state.

**Comment by ekohl** [comment]
https://github.com/theforeman/foreman-packaging/pull/5043 was already merged by @zjhuntin

**Comment by ezr-ondrej** [comment]
/me will wait with this until this repo gets bit more stable... :thinking:

**Comment by ehelms** [comment]
[test rpm]

**Comment by ezr-ondrej** [comment]
false alarm, still broken ... xD

**Comment by evgeni** [comment]
@ehelms uuuh, now f-assets is not installable as foreman still requies < 5.0 until https://github.com/theforeman/foreman/pull/7191 is merged.

**Comment by evgeni** [comment]
@ehelms I did untag the 6.0.1 version for now, so that things can be built again

**Comment by ezr-ondrej** [comment]
> @ehelms I did untag the 6.0.1 version for now, so that things can be built again

Yeah, sorry not to notify it'll fail, but I didn't wanted to unpin, but I still don't know what way it's supposed to be done.

**Comment by ekohl** [comment]
@ezr-ondrej it's good practice to state "depends on ..." in the opening message to indicate they're related. We have a label "After core merge" we can apply. It also makes it easy for reviewers to go back and forth.

**Comment by mmoll** [comment]
Core PR got merged now

**Pull request merged**